### ISSUE TITLE
Group throughput chart data by span action

### ISSUE THREAD

**Issue by gggritso** [getsentry/sentry#71118]
tl;dr

- the "Requests Per Minute" chart on the Requests landing page (and domain summary page?) should be split up by HTTP operation. i.e., 5 lines, for HTTP GET, HTTP POST, etc.
- the "Queries Per Minute" chart on the Queries landing page (and the queries summary page?) should be split by database operation. i.e., 5 lines for SELECT, INSERT, etc.

A few notes:

- we've gotten this request from a few people now, and it makes a lot of sense
- a "Top 5" chart also shows an "Other" category, so we don't need to worry about showing too many lines
- this should probably be a toggle, and maybe a filter, so people can select the exact action they're interested in
- if splitting up by _all_ actions is too granular, we can split up by "theme" e.g., in Databases, read vs. write operations rather than every possible operation
